src/utils/notifications.js

```diff
@@ -155,25 +155,26 @@ export const getToastNotification = (data: mixed, myEthAddress: ?string): null |
   return null;
 };
 
-export const mapInviteNotifications = (notifications: ApiNotification[]): Object[] => notifications
-  .map(({ createdAt, payload }) => {
-    let notification: Object = { createdAt };
-
-    // note: payload.msg is optional and per Sentry reports might not be JSON
-    if (payload.msg) {
-      try {
-        const parsedMessage = JSON.parse(payload.msg);
-        notification = { ...notification, ...parsedMessage };
-      } catch (e) {
-        //
+export const mapInviteNotifications = (notifications: ApiNotification[]): Object[] =>
+  notifications
+    .map(({ createdAt, payload }) => {
+      let notification: Object = { createdAt };
+
+      // note: payload.msg is optional and per Sentry reports might not be JSON
+      if (payload.msg) {
+        try {
+          const parsedMessage = JSON.parse(payload.msg);
+          notification = { ...notification, ...parsedMessage };
+        } catch (e) {
+          //
```

Comment:
We should log something here, unless there was a reason not to?

Reply:
Actually, I have not made changes here. These are auto-save format changes. but I set a log here